include/examples/simple_robot.h

```diff
@@ -0,0 +1,29 @@
+/**
+ * @file simple_robot.h
+ * @author J. Diego Caporale
+ * @brief A simple RobotInterface derived class example. Here is where you would implement any state updates,
+ *        behaviors, or control system
+ * @date 2022-07-19
+ *
+ * @copyright Copyright (c) 2022
+ *
+ */
+
+#pragma once
+
+#include "kodlab_mjbots_sdk/robot_interface.h"
+
+class SimpleRobot : virtual public kodlab::RobotInterface
+{
+
+    using kodlab::RobotInterface::RobotInterface;
+
+public:
+    int mode = 0; // Member variables encode whatever added state we need
+    // Set up robot update function for state and torques
+    void Update() override
+    {
+        std::vector<float> torques(num_joints_, 0);
```

Comment:
I think this is fine, but we really want to make the SimpleRobot demonstrate more about how we expect the `RobotInterface` to be used.

Reply:
Agreed, we should do a formatting run (see #35).

Reply:
Kept in issue https://github.com/KodlabPenn/kodlab_mjbots_sdk/issues/36#issue-1326662355 
Will make a PR just for this change.